actions/campaigns.go

```diff
@@ -381,3 +382,27 @@ func GetCampaignOpens(c *gin.Context) {
 		"message": "Id must be an integer",
 	})
 }
+
+func GetCampaignClicksStats(c *gin.Context) {
+	id, err := strconv.ParseInt(c.Param("id"), 10, 64)
+	if err != nil {
+		c.JSON(http.StatusBadRequest, gin.H{
+			"message": "Id must be an integer",
+		})
+	}
+
+	stats, err := storage.GetCampaignClicksStats(c, id, middleware.GetUser(c).ID)
+	if err != nil {
+		c.JSON(http.StatusNotFound, gin.H{
+			"message": "Campaign clicks not found",
+		})
+		return
+	}
+
+	c.JSON(http.StatusOK, entities.CampaignClicksStats{
+		Total:       int64(len(stats)),
+		ClicksStats: stats,
+	})
+	return
```

Comment:
Omit the return here, staticcheck fails for redundant returns.

Reply:
@FilipNikolovski i removed the return here so i am resolving this too